src/Raven.Server/Documents/Patch/JavaScriptUtils.cs

```diff
@@ -400,7 +398,9 @@ internal JsValue TranslateToJs(Engine engine, JsonOperationContext context, obje
 
             if (o is BlittableJsonReaderObject json)
             {
-                return new BlittableObjectInstance(engine, null, Clone(json, context), null, null, null);
+                // check if clone is really required, we don't want to clone patch arguments
+                BlittableJsonReaderObject blittable = needsClone ? Clone(json, context) : json;
```

Comment:
@garayx please review, the only thing that worries me is that this blittable might come from a different context and I do not know how long this context is alive